docs/samples/Microsoft.ML.Samples/Dynamic/Normalizer.cs

```diff
@@ -1,8 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
 using Microsoft.ML.Data;
-using Microsoft.ML.Transforms;
+using Microsoft.ML.Experimental;
```

Comment:
Is it just experimental? We are removing the generic estimator, so these new methods must be supported officially and therefore deserve a namespace like `ML.Transform`. Is my understanding correct?

Reply:
https://github.com/dotnet/machinelearning/pull/3118